example/src/domain/machinery/Form.js

```diff
@@ -34,6 +34,42 @@ export function FormCheckbox({ field, label }) {
   )
 }
 
+export function FormCheckboxGroupField({ field, options, label }) {
+  const { name, state, eventHandlers: { onChange, ...eventHandlers } } = useFormField(field)
+  const { value } = state
+
+  console.log(`[${name}] render checkbox group field`)
+
+  return (
+    <FieldsetAndError {...{ label, state }}>
+      {options.map((option, i) => (
+        <div key={`${name}__${i}`}>
+          <label htmlFor={name}>{option.label}</label>
+          <input
+            id={`${name}__${i}`}
```

Comment:
`htmlFor` en `id` moeten overeen komen.

```suggestion
      {options.map((option, i) => {
        const id = `${name}__${i}`
        return (
          <div key={id}>
            <label htmlFor={id}>{option.label}</label>
            <input
              {...{ id } }
```